nrf_security/src/CMakeLists.txt

```diff
@@ -122,6 +122,7 @@ append_with_prefix(src_tls ${ARM_MBEDTLS_PATH}/library
     ssl_ticket.c
     ssl_tls.c
     ssl_tls13_keys.c
+    ssl_debug_helpers_generated.c
```

Comment:
I can't see this file.

I think this patch needs to be rebased after the Mbed TLS 3.2.1 update.

Reply:
I need this to have `mbedtls_ssl_states_str` defined